### PR TITLE
[M] ENT-2725: Update system purpose status to NOT consider Layered SLA

### PR DESCRIPTION
**Pull request by sonalidhome**

### CODE REVIEW

File: server/spec/autobind_spec.rb [hunk 1]
```diff
@@ -822,4 +822,38 @@
...
+    @cp.consume_product(product2['id'], {:uuid => consumer.uuid})
+
+    status = @cp.get_purpose_compliance(consumer['uuid'])
+    expect(status['status']).to eq('not specified')
+
+    entitlements = @cp.list_entitlements(:uuid => consumer.uuid)
+
+    entitlements.each do |ent|
+      @cp.unbind_entitlement(ent.id, {:uuid => consumer.uuid})
+    end
```
Comment (nikosmoum) [inline server/spec/autobind_spec.rb:856]: The final check here is missing

**Review by nikosmoum — CHANGES_REQUESTED** [review]
1 minor issue; otherwise good

**Comment by nikosmoum** [comment]
Also, please avoid using the flags like [M] in the commit messages. We only add it on the PR summary for convenience

File: server/spec/autobind_spec.rb (hunk 1 above)
Comment (sonalidhome) [reply server/spec/autobind_spec.rb:856]: Updated.

**Pull request merged**